code/modules/research/rdconsole.dm

```diff
@@ -162,17 +162,10 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 			KT.level=KT.max_level
 	files.RefreshResearch()
 
-/obj/machinery/computer/rdconsole/New()
-	..()
+/obj/machinery/computer/rdconsole/Initialize(mapload)
+	. = ..()
 	files = new /datum/research(src) //Setup the research data holder.
 	matching_designs = list()
-	if(!id)
-		for(var/obj/machinery/r_n_d/server/centcom/S in GLOB.machines)
-			S.initialize_serv()
-			break
```

Comment:
This is a scary part. I couldn't figure out what it's supposed to be doing.

At the time of the /New, the `GLOB.machines` is empty, so the for loop does nothing (i checked).
And it's linking to centcom servers and not station ones?
Except it's not even *linking* anything - it's only just *initializes* it.
Basically i'm very confused, and consider this to be legacy code that is better off dead.
Rnd console and its sync with the server works fine without it.

There's also this curious line at the end of rnd server /New, which... looks vaguely significant
https://github.com/ParadiseSS13/Paradise/blob/0716543f32a362e1a7035955e77a7ccf54d0ea6c/code/modules/research/server.dm#L18-L26